pgModel/models.go

```diff
@@ -0,0 +1,64 @@
+package pgModel
+
+type JwtToken struct {
+    Token string `json:"token"`
+}
+
+type Exception struct {
+    Message string `json:"message"`
+}
+
+type User struct {
+	Id     int64 `sql:",pk,unique"`
+	Username   string
+	Email string `sql:",unique,notnull"`
+	Password string
+	PhoneNumber string
+	Messages []*Message
+	Connections []*Connection
+	Comments []*Comment
+	Collections []*Collection
+}
+
+type Connection struct {
+	Id int64
+	FirstUserId int64
+	SecondUserId int64
+}
+
+type Message struct {
+	Id int64
+	UserId int64
+	ReceiverId int64
+	Content string
+}
+
+type Resource struct {
```

Comment:
@mikey2020 great work on adding Link to the Resource model.
However the title, privacy and UserId fields are omitted.
These are represented in the relational schema.

Reply:
Also, I think tags should be added

Reply:
You said we should hold on on the tags relationship